lib/elixir/src/elixir_utils.erl

```diff
@@ -33,6 +33,14 @@ get_line(Opts) when is_list(Opts) ->
     false -> 0
   end.
 
+get_generated(Opts) when is_list(Opts) ->
+  case erlang:system_info(otp_release) of
+    Release when Release >= "19" ->
```

Comment:
Struct type specs are compiled differently on OTP 19 and 18. In the past there have also been incompatibilities when elixir was compiled against an OTP version different than what it is running on.